pkg/storage/migrator/migrator.go

```diff
@@ -34,6 +35,7 @@ func NewMigratorConfig() *MigratorConfig {
 		Unlocker_handler: func() {},
 		Error_handler:    func(b *storage.BlockInfo, err error) {},
 		Progress_handler: func(p *MigrationProgress) {},
+		Block_handler:    func(b *storage.BlockInfo, id uint64, data []byte) {},
```

Comment:
We really should be using snake case in Go

Reply:
https://go.dev/doc/effective_go#mixed-caps suggests the convention is camelCase now?

I agree we should settle on one. I keep slipping up. However, this would be a breaking change...
We should put together a lint config for this sort of thing. I can't find a lint setting related to camelCase/snake_case yet though do you know of one?

Reply:
Agree... this will be a breaking change though, so I will fix all my snake_case slip ups in a separate PR